DcCore/DcCore/Extensions/String+Extensions.swift

```diff
@@ -2,8 +2,12 @@ import Foundation
 import UIKit
 
 public extension String {
-    
-	static func localized(_ stringID: String) -> String {
+
+    static func markAsExternal(_ string: String) -> String {
+        return string + " ↗"
```

Comment:
isn't a non static func easier to use?
```suggestion
    func markAsExternal() -> String {
        return self + " ↗"
```
then you can use it like:
```diff
- String.markAsExternal(String.localized("example"))
+ String.localized("example").markAsExternal()
```